rizinorg/rizin
# Added 'aecb' to continue backwards in ESIL ##trace

**Your checklist for this pull request**
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/rizinbook) with the relevant information (if needed)

**Detailed description**

We only had `aesb` to step backwards, added `aecb` to return to the nearest breakpoint/start of session. Tried to stay consistent with `dcb`'s implementation using `esil_trace`.

**Test plan**

* Start a session with: `aei; aeim; aeip; aets+`
* Step forward with: `aes`
* Continue backwards with: `aecb`
* Check that you have returned to the nearest breakpoint/first index with the previous state correctly restored.
